setup.py

```diff
@@ -27,7 +27,7 @@ def read(fname):
     long_description=read('README.md'),
     entry_points={
         'console_scripts': [
-            "paper_git = paper_to_git.bin.paper_git:main",
+            "paper-git = paper_to_git.bin.paper-git:main",
```

Comment:
Since the module name doesn't change, this is also supposed to be just `paper_git`.
It would look something like:
```paper-git` = paper_to_git.bin.paper_git:main```